db/client.go

```diff
@@ -0,0 +1,38 @@
+package db
+
+import (
+	"EmployeeManagementSystem/models"
```

Comment:
The import shouldn't be done like that.. It won't work with wondering.

It should be 

`"github.com/ahsanfayaz52/EmployeeManagementSystem/models"